.github/workflows/release.yml

```diff
@@ -18,6 +18,18 @@ jobs:
       - name: Get the version
         id: get_version
         run: echo ::set-output name=VERSION::$(git describe --tags --abbrev=0 HEAD)
+      - name: Get the ChangeLog
+        id: get_changelog
+        shell: bash
+        run: |
+          changelog=$(grep -B100 -m2 -- '----' CHANGELOG.md)
+          changelog="${changelog//'%'/'%25'}"
+          changelog="${changelog//$'\n'/'%0A'}"
+          changelog="${changelog//$'\r'/'%0D'}"
+
+          echo "$changelog"
```

Comment:
Why print this? Since we get the output in the release message body, I don't see the benefit of this